modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -271,22 +275,21 @@ The record payloads for random access are exposed as an `immutable List`, and si
 
 === Batch Aggregator Characteristics
 
-* The batch aggregator scope can only exist in batch steps which, in turn, are only usable within the batch process phase of a batch job. You cannot use batch aggregators during the On Complete phase of a batch job.
-* An aggregator can only wrap the final element within the batch step in which it resides.
-* Several *Anypoint Connectors* can handle record-level errors without failing a whole batch aggregation (i.e., upsert). +
+* The Batch Aggregator scope can only exist in a Batch Step which, in turn, is only usable within the batch process phase of a Batch Job. You cannot use Batch Aggregator components during the On Complete phase of a Batch Job.
+* A Batch Aggregator can only wrap the final element within the Batch Step in which it resides.
+* Several Anypoint connectors can handle record-level errors (such as, `upsert` errors) without failing a whole batch aggregation.
 At runtime, these connectors keep track of which records were successfully accepted by the target resource, and which failed to upsert. Thus, rather than failing a complete group of records, the connector upserts as many records as it can, and tracks any failures for notification. Some of these connectors are:
 +
 ** Salesforce
 ** NetSuite
 ** Database
 +
 To make sure that the connector you are using supports record-level errors, check the connector's documentation.
-* The batch aggregator scope does not support job-instance-wide transactions. You can define a transaction inside a batch step that processes each record in a separate transaction. Think of it as a step within a step. +
+* The Batch Aggregator scope does not support job-instance-wide transactions. You can define a transaction inside a Batch Step that processes each record in a separate transaction. Think of it as a step within a step.
```

Comment:
this is incorrect. It's not aggregator that doesn't support instance wide transactions. It's batch itself that doesn't support it. If you open a transaction inside a batch step, that transaction will only cover that particular record, on that particular step, and there will be as many transactions as records go through that step